Readability.js

```diff
@@ -1263,26 +1267,25 @@ Readability.prototype = {
    * @return void
   **/
   _cleanStyles: function(e) {
-    e = e || this._doc;
-    if (!e)
+    if (!e || e.tagName === 'svg')
```

Comment:
It looks like in jsdom, tagName is "svg" and in JSDOMParser it's "SVG".  Lowercase is correct (or at least, that's what gecko does too), but it seems we need to call `.toLowerCase()` to make it work in JSDOMParser. :-(